packages/react-components/src/MessageSignature.tsx

```diff
@@ -2,11 +2,11 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { ContractABIMessage } from '@polkadot/api-contract/types';
+import { AbiMessage } from '@polkadot/api-contract/types';
 
 import React from 'react';
 import styled from 'styled-components';
-import { displayType } from '@polkadot/types';
+import { encodeTypeDef } from '@polkadot/types';
```

Comment:
Can we use @cennznet/types here instead?

Reply:
yes..we can... My initial plan is to make it work.. There are many places through out the code base where @polkadot packages are used.. we only replaced the API object with CENNZnet API so that all the extrinsics, query storage is available... But I think we can have a separate task for this.. 
can be one of the TODO's to replace types with cennznet types..